app/models/storage_profile.rb

```diff
@@ -0,0 +1,5 @@
+class StorageProfile < ApplicationRecord
+  belongs_to :ext_management_system
+  has_many :storage_profile_storages, :dependent  => :destroy
```

Comment:
[Here](https://github.com/jameswnl/manageiq/blob/sp-test/app/models/storage_profile.rb#L4) is how I did the private relation